test/spec/features/modeling/ReconnectConnectionSpec.js

```diff
@@ -212,6 +241,23 @@ describe('features/modeling - reconnect connection', function() {
         expect(connection.waypoints).to.eql(oldWaypoints);
       }));
 
+
+      it('should layout connection', inject(function(modeling) {
```

Comment:
I like that you added these explicit layout tests :tada:.

What about the `redo` case though? It would have made clear that the reconnect handler does not properly revert things on undo :wink:.